source/module_esolver/esolver_ks.h

```diff
@@ -72,6 +75,10 @@ class ESolver_KS : public ESolver_FP
         //! <Temporary> It should be replaced by a function in Hamilt Class
 		virtual void update_pot(const int istep, const int iter) {};
 
+        virtual double Run_rdmft(ModuleBase::matrix& E_gradient_occNum, psi::Psi<T>& E_gradient_wfc) { return 0.0; };   // add by jghan, 2024-03-16
```

Comment:
We have a convention of naming classes, functions and variables. For classes, it is recommended to use something like `ReducedDensityMatrixFunctionalTheory`, for functions and variables, use `reduced_density_matrix_functional_theory` or abbr of it like `rdmft_*`. The function name `update_elec_rdmft` is a good practice. :)

Reply:
ok, I  will use run_rdmft() to replace